src/DynamoCore/Graph/Workspaces/WorkspaceInfo.cs

```diff
@@ -49,6 +49,7 @@ public WorkspaceInfo()
                 double cx = 0;
                 double cy = 0;
                 double zoom = 1.0;
+                double scaleFactor = 1.0;
```

Comment:
We have zoom property serialized to workspace db, currently I believe we have no case utilizing the scaleFactor, but if we want to have this ASM scaling in Flood editor then we will probably need this